### ISSUE TITLE
Update cimplot pin

### ISSUE THREAD

**Issue by Fuuzetsu** [4bb4/implot-rs#19]
There is some new functionality like labels for second and third Y axis in BeginPlot.

**Comment by 4bb4** [comment]
I'm trying this out in https://github.com/4bb4/implot-rs/tree/19-update-cimplot-pin, but I'm having a bit of difficulty with it since cimplot changed its structure around a bit. Will report when I make progress.

**Comment by 4bb4** [comment]
It turns out this is blocked upstream by imgui-rs not yet using the very latest cimgui - and the latest cimplot requires the latest cimgui (see https://github.com/cimgui/cimplot/issues/8). I'll wait a bit to see if this comes along soon. Otherwise we could consider adding cimplot as a direct submodule and not depending on imgui-rs anymore for this, but that might jeopardize interoperability with implot-rs so I'm hesitant to go that route.

**Comment by Fuuzetsu** [comment]
Is there some specific tracking issue we have to wait for? There's https://github.com/imgui-rs/imgui-rs/issues/462 but that's not looking like it's going to happen any time soon.

Technically I think they could update the cimgui and publish new imgui-sys even if they don't have nice wrappers for it in imgui-rs... but I'm unsure, not too involved with this stuff.

**Comment by 4bb4** [comment]
> Technically I think they could update the cimgui and publish new imgui-sys

Yes, it seems that way, though the current maintainer (rightfully) has concerns about semver compatibility of the internal APIs - there is an issue in https://github.com/imgui-rs/imgui-rs/issues/395 already. I've commented there to see where things stand, and also found out that I can (temporarily) create a version of cimplot with the internal API disabled if the aforementioned issue takes longer to resolve.
